apps/admin-x-framework/src/api/invites.ts

```diff
@@ -57,8 +80,12 @@ export const useDeleteInvite = createMutation<unknown, string>({
         dataType,
         emberUpdateType: 'delete',
         update: (_, currentData, id) => ({
-            ...(currentData as InvitesResponseType),
-            invites: (currentData as InvitesResponseType).invites.filter(invite => invite.id !== id)
+            ...(currentData as {pages: InvitesResponseType[]}),
+            pages: (currentData as {pages: InvitesResponseType[]}).pages.map((page: InvitesResponseType) => ({
+                ...page,
+                invites: page.invites.filter(invite => invite.id !== id)
+            }))
```

Comment:
Similar to the above comment, we have a [`deleteFromQueryCache` helper](https://github.com/TryGhost/Ghost/blob/main/apps/admin-x-framework/src/utils/api/updateQueries.ts#L62-L86) that looks like it could be used here.

I could be wrong as I'm not particularly context loaded on any Admin-X stuff, it's just that the manual updates stuck out when comparing against other infinite query uses.